__WIP_pg_schema_review/readme.md

```diff
@@ -0,0 +1,47 @@
+# If you only read one thing, read the `provision.sql` file
+
+# SQL Migrataion scratchpad Repo
+
+This is a repo to test out some ideas for migrating and defining the sql schemas in the postgres database that will be the target for the migration away from fauna
+
+At this moment (8/30/2021) there's just a few folders
+
+- niftysave- Literally just a copy of the niftysave repo so its nearby and convenient to look at
```

Comment:
This is gone now

```suggestion
```

Reply:
sorry this should be deleted, I forgot to remove it on my new branch

Reply:
yes I'll move it into the databases section